### ISSUE TITLE
[BITV] 9.4.1.2/3.2 - The "Actions" menu buttons in the table should be implemented using the aria-haspopup="menu" attribute to inform users that a menu is being opened. (2)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41890]
https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v3-n2

9.4.1.2/11.2 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v11-n2
